src/main/java/org/osiam/client/OsiamConnector.java

```diff
@@ -799,6 +811,8 @@ public QueryBuilder createQueryBuilder(Query original) {
         private String authServiceEndpoint;
         private String resourceServiceEndpoint;
         private String clientRedirectUri;
+        private Integer connectTimeout;

```

Comment:
It might be a good idea to keep the type of the parameter the same in all classes either the primitive or the Integer.